cloud_kafka/librdkafka/config_parser.h

```diff
@@ -0,0 +1,65 @@
+#ifndef CONFIG_PARSER_H
+#define CONFIG_PARSER_H
+
+#include <fstream>
+#include <map>
+#include <string>
+
+class ConfigParser {
```

Comment:
You don't need to write the parse logic manually. Just use an existing library like https://github.com/benhoyt/inih?tab=readme-ov-file#c-example, which is available in vcpkg and easy to use.

Reply:
Let me take over this PR